api/v1/rbac_manager_service.proto

```diff
@@ -1,8 +1,8 @@
 syntax = "proto3";
 
-package llmoperator.rbac.server.v1;
+package llmariner.rbac.server.v1;
```

Comment:
This change should be ok, but we need to be careful. The package path is in the ingress (e.g., https://github.com/llmariner/job-manager/blob/main/deployments/server/templates/ingress.yaml#L68) so we might need to keep the old path for backward compatibility.